specification/implementation/metadata/general-requirements/index.md

```diff
@@ -1,66 +1,65 @@
 case of SIPs, appropriate preservation metadata is not always available. As such this is also not an absolute requirement though highly desirable. The detailed specification of the use of PREMIS within the CSIP is available in Section 5.4.
 
-The use of any additional metadata is not restricted in CSIP Information Packages.
+The specification does not prohibit the use of additional metadata within Information Packages.
 
 ## 5.2 General requirements for the use of metadata
-Before we describe the detailed requirements for the use of METS and PREMIS we would like to highlight some general aspects which need to be implemented commonly across all metadata.
+Before detailing the requirements covering the use of METS and PREMIS we would like to highlight some general metadata concerns which should be handled consistently.
 
 ### The use of identifiers
-The ID data type in XML (<https://www.w3.org/TR/xml-id/>) states that a valid ID must begin with a letter, or an underscore character (‘_’), and contain no characters other than letters, digits, hyphens, underscores, full stops, and certain combining and extension characters. To overcome this limitation and allow for interoperable package identification all identifiers within Common Specification metadata MUST start with a prefix, followed by the value of the identifier.
+Both METS, and by derivation the CSIP, make extensive use of the XML ID data type, see: <https://www.w3.org/TR/xml-id/>. Valid XML IDs must also conform to the NCName restrictions, see: <https://www.w3.org/TR/REC-xml-names/#NT-NCName>, and must begin a letter or an underscore character (‘_’), and contain no characters other than letters, digits, hyphens, underscores, full stops, plus some combination and extension characters. If your package makes use of IDs that don't conform to this specification one mitigating strategy is to add a prefix to all identifiers. For example UUIDs are a common choice for arbitrary identifiers and comprise of hex characters and hyphens, e.g. the value `906F4F12-BA52-4779-AE2C-178F9206111F`. NCName and xml:id values can't legally begin with numeric characters making many UUIDs,
+including the example, invalid. The examples below show two possible solutions, using prefixes:
 
 Examples:
 
-Example 1: using a prefix which consists of the abbreviation of the identifier and a hyphen.
+Example 1: The prefix consists of the identifier type acronym and a hyphen: `uuid-`.
 
 ```xml
 <dmdSec ID="uuid-906F4F12-BA52-4779-AE2C-178F9206111F" CREATED="2018-04-
 24T14:37:49.609+01:00">
 ```
 
-Example 2: using a fixed prefix “ID”
+Example 2: A fixed prefix:`ID`.
```

Comment:
```suggestion
Example 2: A generic prefix: `ID`.
```

`uuid-` in example 1 is also fixed, so we need another word, "generic" is the first that came to mind.